sale_unreconciled/models/sale_order.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2021 ForgeFlow S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.osv import expression
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    unreconciled = fields.Boolean(
+        compute="_compute_unreconciled",
+        search="_search_unreconciled",
+        help="""Indicates that a sale Order has related Journal items not
+             reconciled.Note that if it is false it can be either that
+             everything is reconciled or that the related accounts do not
+             allow reconciliation""",
+    )
+    is_delivered = fields.Boolean(search="_search_is_delivered")
```

Comment:
I don't know if this feature is adequate in this module, because I would understand that it has more to do with the stock than with the reconciliation of the sales order.

Reply:
The thing is, this field was part of a previous Odoo version and it was used by this module. But I agree it should be out of this module.